server/methods/projects.js

```diff
@@ -37,5 +38,14 @@ Meteor.methods({
   'projects/sync': function () {
     if (!this.userId) throw new Meteor.Error('Only a logged in user can sync projects')
     return syncAllAsync(this.userId)
+  },
+
+  'projects/tag': function (projectId) {
+    if (!this.userId) throw new Meteor.Error('Only a logged in user can tag a project')
+
+    let project = Projects.findOne(projectId)
+    if (!project) throw new Meteor.Error('Cannot find project')
+
```

Comment:
Belt and braces I suppose yes. Needless to say they won't be able to create a tag in it if they don't have access.